### DISCUSSION TITLE
IfcConvert : How to render in svg IfcStair / IfcStairFlight

### DISCUSSION THREAD

**Discussion by DaiNguyenTM** [IfcOpenShell/IfcOpenShell#2635] [Q&A] [answered]
Hello all,
It is possible to render IfcStair / IfcStarFlight in svg by IfcConverter ?
![image](https://user-images.githubusercontent.com/119857165/205917845-3a404bfd-5de0-43b0-a54d-7d08207f14cd.png)

Thank you.

**Comment by aothms** [comment, marked as answer]
Not in the way you show in your image. We have annotations for door arcs with `--door-arcs` but stair walking lines we haven't specifically implemented yet. If you use `--svg-project` you will get a hidden line rendering output of the stairs of solely it's projected geometry.
